components/channel_header_dropdown/menu_items/view_and_manage_members.js

```diff
@@ -34,28 +40,27 @@ const ViewAndManageMembers = ({channel, isDefault}) => {
                 dialogProps={{channel}}
             >
                 {!isDefault &&
-                    <ChannelPermissionGate
-                        channelId={channel.id}
-                        teamId={channel.team_id}
-                        permissions={[permission]}
-                    >
-                        <FormattedMessage
-                            id='channel_header.manageMembers'
-                            defaultMessage='Manage Members'
-                        />
-                    </ChannelPermissionGate>
-                }
-                <ChannelPermissionGate
-                    channelId={channel.id}
-                    teamId={channel.team_id}
-                    permissions={[permission]}
-                    invert={!isDefault}
-                >
-                    <FormattedMessage
-                        id='channel_header.viewMembers'
-                        defaultMessage='View Members'
-                    />
-                </ChannelPermissionGate>
+                    <>
+                        <ChannelPermissionGate
+                            channelId={channel.id}
+                            teamId={channel.team_id}
+                            permissions={[permission]}
+                        >
+                            <FormattedMessage
+                                id='channel_header.manageMembers'
+                                defaultMessage='Manage Members'
+                            />
+                        </ChannelPermissionGate>
+                        <ChannelPermissionGate
+                            channelId={channel.id}
+                            teamId={channel.team_id}
+                            permissions={[permission]}
+                            invert={!isDefault}
```

Comment:
Now this is inside the `!isDefault` conditional, this line could be imo a little clearer as:

```suggestion
                            invert={true}
```